src/hooks/useRecordingTrigger.ts

```diff
@@ -43,7 +40,11 @@ export default function useRecordingTrigger(
       event.preventDefault()
       event.stopPropagation()
 
-      const HIDcode = webCodeHIDLookup.get(event.code)?.HIDcode
+      // Gets the ID according to the whichID, adds a separator extra digit '1' and then adds location to the end.
+      const HIDIdentifier = event.which + '1' + event.location
+
+      const HIDcode = webCodeLocationHIDLookup.get(HIDIdentifier)?.HIDcode
+
```

Comment:
There's a few places where it's doing this encoding, would be good to make it a function so that they all share the same encoding logic

Reply:
Extracted to a function here: [109c888](https://github.com/WootingKb/wooting-macros/pull/194/commits/109c8883e67e4844e29ee45f305aa2d6547ade38)